### PR TITLE
Added missing datepicker options pickDate. Considered the possibility in the template to display only timepick.

### PR DESCRIPTION
**Pull request by Fuck4ik**

### CODE REVIEW

**Comment by greg0ire** [comment]
Please associate your gmail email address with your Github account, or change the email in your commits to an address already associated with it.

**Comment by Fuck4ik** [comment]
Done

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "sonata-project/core-bundle",
+    "name": "omasn/core-bundle",
```
Comment (greg0ire) [inline composer.json]: Why would you do that?

Comment (jlamur) [reply composer.json]: Reverted in the last commit.

**Comment by greg0ire** [comment]
Please get rid of the first and last commits by squashing or rebasing, then force pushing, also,

please format your commit message according to our rules : 

- The commit message subject must be less than 50 characters and tell us *what* you did.
- The commit message body should tell us *why* you did it. It is optional but highly recommended.

Bad example :

```
Fixed bug #989 by removing call to defraculate()
```

Good example:

```
Remove call to defraculate()

Calling this function caused a bug because it interferes 
with calls to getSchmeckles().
Fixes #989
```

More details in CONTRIBUTING.md

File: Resources/views/Form/datepicker.html.twig [hunk 2]
```diff
@@ -39,12 +39,14 @@ file that was distributed with this source code.
 
 {% block sonata_type_datetime_picker_widget_html %}
     {% if datepicker_use_button %}
-        <div class='input-group date' id='dtp_{{ id }}'>
+        <div class='input-group date {% if not dp_options['pickDate'] %}timepicker{% endif %}' id='dtp_{{ id }}'>
```
Comment (jlamur) [inline Resources/views/Form/datepicker.html.twig]: @greg0ire WDYT of `if dp_options['pickDate'] is defined and not dp_options['pickDate']` for BC?

Comment (greg0ire) [reply Resources/views/Form/datepicker.html.twig]: would be better indeed

**Comment by Fuck4ik** [comment]
There are no cases when this option is absent

**Pull request closed without merging**